src/parsing/parsers/generic/jison/sql_main.test.json

```diff
@@ -34,5 +34,58 @@
     "expectedResult": {
       "lowerCase": false
     }
+  },
+  {
+    "namePrefix": "should contain suggestSnippets",
+    "beforeCursor": "",
+    "afterCursor": "",
+    "containsKeywords": [
+      "ALTER",
+      "CREATE",
+      "DELETE",
+      "DESCRIBE",
+      "DROP",
+      "EXPLAIN",
+      "GRANT",
+      "INSERT",
+      "REVOKE",
+      "SELECT",
+      "SET",
+      "SHOW",
+      "TRUNCATE",
+      "UPDATE",
+      "USE",
+      "WITH"
+    ],
+    "expectedResult": {
+      "lowerCase": false,
+      "suggestSnippets": true
+    }
+  },
+  {
+    "namePrefix": "should contain suggestSnippets with EXPLAIN prefix",
+    "beforeCursor": "EXPLAIN ",
+    "afterCursor": "",
+    "containsKeywords": [
+      "ALTER",
+      "CREATE",
+      "DELETE",
+      "DESCRIBE",
+      "DROP",
+      "GRANT",
+      "INSERT",
+      "REVOKE",
+      "SELECT",
+      "SET",
+      "SHOW",
+      "TRUNCATE",
+      "UPDATE",
+      "USE",
+      "WITH"
+    ],
+    "expectedResult": {
+      "lowerCase": false,
+      "suggestSnippets": true
+    }
   }
```

Comment:
Let's create a test that will check that there's `suggestSnippets: undefined` when we've written `SELECT * FROM |` for example

Reply:
JSON haven't `undefined` value type, we can set `suggestSnippets: false` and in this case we will check that `parser result` should not contain this key.

Reply:
Good, let's do it then!